dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -371,6 +381,30 @@ def _load_hist_helper(self, data):
                 }
             self._stored_histogram[key] = value
 
+        # Convert values to correct types
```

Comment:
do we not need to also update: `self.histogram_methods`?

Reply:
I dont believe so, the tests have passed with current typing.